client/app/components/api/ApiView.jsx

```diff
@@ -0,0 +1,84 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+var RouteHandler = Router.RouteHandler;
+var DocumentActions = require("../../actions/DocumentActions.js");
+var DocumentStore = require('../../stores/DocumentStore.js');
+var ReactBootstrap = require('react-bootstrap'),
+    Col = ReactBootstrap.Col,
+    Row = ReactBootstrap.Row,
+    Grid = ReactBootstrap.Grid;
+
+// Section Component
+var Section =  React.createClass({
+
+  render: function () {
+    //element array
+    var links = [];
+    for (var i = 0; i<this.props.section.length; i++){
+      //push doc link into links
+      links.push(<div><Link to="api.section.document" params={{section: section, document: i}}>{i}</Link><br/></div>);
+    }
+    return (
+      <div>
+        //section link
```

Comment:
Remove comments within the render return statement. There might be issues with JavaScript comments in HTML code.